### DISCUSSION TITLE
[Projects Beta] Reviewers column like assignee column

### DISCUSSION THREAD

**Discussion by achimnol** [community/community#8164] [Projects and Issues] [answered]
I'd like to filter and group by reviewers as well as assignees of the PRs.
If the new projects becomes able to show linked PRs for each issue, the reviewers field may be an aggregation of all linked PRs as we wouldn't add PRs with linked issues to the projects to avoid duplication.

**Comment by achimnol** [comment, marked as answer]
This is now available after [the Jan 13 update](https://github.blog/changelog/2022-01-13-the-new-github-issues-01-13-update/). 🎉

**Reply by alirezaayande** [reply]
Hi, I have a very new project, I am just looking for a foreign sponsor and business partner, and I am sure it is great in my area. Can you help me? The plan is about betting sites Direct by email:alirezaayande@gmail.com Call thank you

**Comment by andrewalexanderbalogh** [comment]
I finally understand how the **_Reviewers_** column works on the Table view after having read the answer by @kevinwhoffman 
~ here https://github.com/community/community/discussions/10300?sort=top

With our companies projects, in our Table View we just include Links to **Issues**, we've  never included Links to PR's directly.
For us, the display of the _**Linked Pull Requests**_ column is sufficient where PR's should be shown, and inclusion of every PR would just add row clutter.

That said, in our case we have always had to create _Custom Single-Select_ columns with an option for each Reviewer
in order to show the Reviewer associated with a given Issue.

I'd like to request the option to display associated Reviewers to PR's within the _**Linked Pull Requests**_ column

![image](https://user-images.githubusercontent.com/17296403/224585329-b9816dd3-9545-4fa5-aa2f-ee5e96d14036.png)

**Reply by laymonage** [reply]
Related discussion: https://github.com/orgs/community/discussions/10300
